acq4/devices/geometries/sensapex_lefthanded.srdf

```diff
@@ -0,0 +1,34 @@
+<robot name="linear_actuator_robot">
```

Comment:
Maybe remove these? Or collect all of the almost-pinocchio stuff into one place?